raven/src/main/webapp/META-INF/context.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8'?>
+<Context>
+
+    <WatchedResource>WEB-INF/web.xml</WatchedResource>
+
+    <Resource name="jdbc/inventory"
+        auth="Container"
+        type="javax.sql.DataSource"
+        factory="org.apache.tomcat.jdbc.pool.DataSourceFactory" closeMethod="close"
+        minEvictableIdleTimeMillis="60000" timeBetweenEvictionRunsMillis="30000"
+        maxWait="20000"
+        initialSize="0" minIdle="0" maxIdle="${raven.invadm.maxActive}" maxActive="${raven.invadm.maxActive}"
```

Comment:
invadm isn't really correct here - raven needs read only access to the inventory database.

maybe "invuser"?